CppImport/src/manager/TranslateManager.h

```diff
@@ -131,7 +132,11 @@ class CPPIMPORT_API TranslateManager
 		 */
 		OOModel::TypeAlias* insertTypeAliasTemplate(clang::TypeAliasTemplateDecl* typeAliasTemplate);
 
+		void mapAst(clang::Stmt* clangAstNode, Model::Node* envisionAstNode);
```

Comment:
You have added these two methods to the h file, but they are not defined anywhere. Either define and use them, or remove them.